sys/include/net/sock.h

```diff
@@ -144,6 +144,7 @@ extern "C" {
  * @{
  */
 #define SOCK_FLAGS_REUSE_EP     (0x0001)    /**< allow to reuse end point on bind */
+#define SOCK_FLAGS_BIND_REMOTE  (0x0002)    /**< restrict responses to remote address */
```

Comment:
Isn't the "binding" of a remote typically a `connect`? Given that there are relations between `sock` and the POSIX socket API, using the wording of "bind" might be confusing to the reader here.

```suggestion
#define SOCK_FLAGS_CONNECT_REMOTE   (0x0002)    /**< restrict responses to remote address */

Reply:
fine with me